satellite/console/service.go

```diff
@@ -602,7 +605,8 @@ func (s *Service) GetCurrentRewardByType(ctx context.Context, offerType rewards.
 		s.log.Error("internal error", zap.Error(err))
 		return nil, ErrConsoleInternal.Wrap(err)
 	}
-	return offers.GetActiveOffer(offerType, "")
+
+	return s.partners.GetActiveOffer(ctx, offers, offerType, "")
```

Comment:
Feels odd to have GetActiveOffer be in partners service but it's able to to get other type of offers

Reply:
Yeah, I'm aware. The reason is that the `GetActiveOffer` needs access to the partners database. But, yeah, currently just moved things around to make it work, rather than refactoring to something that feels correct.